src/test/java/jUnit/JsonParserTest.java

```diff
@@ -49,25 +56,28 @@ public void readFromFileTest() {
     void notSuchFileExceptionTest(String path) {
         Exception exception = assertThrows(NoSuchFileException.class,
                 () -> parser.readFromFile(new File(path)));
-        assertEquals("File " + path + ".json not found!", exception.getMessage());
+        assertEquals("File " + path + ".json not found!", exception.getMessage()); //the first test should be fail because path is correct
     }
 
     static Stream<String> getPath() {
-        return Stream.of("src\\main\\resources\\vadim-cart", "vadim-cart.json", "vadim-cart", "resources\\vadim-cart", "vadim cart");
+        return Stream.of("src\\main\\resources\\andrew-cart.json", "vadim-cart.json", "vadim-cart", "resources\\vadim-cart", "vadim cart");
     }
 
     @Disabled
```

Comment:
as per task requirements that's correct, but practically this test would be useless, so remove the test and feel free to disable any other test in the same way